src/AirtableClient.php

```diff
@@ -110,7 +110,7 @@ public function findTheLatest(string $table, $field, ?string $dataClass = null):
     /**
      * {@inheritdoc}
      */
-    public function addOneRecord(string $table, array $fields, ?string $dataClass = null): ?AirtableRecord
+    public function add(string $table, array $fields, ?string $dataClass = null): ?AirtableRecord
```

Comment:
Wait wait, I know this is not the point of this PR but .. This method (add) should take $fields as an array OR an object which would be denormalized as an array no ?

Reply:
True... So, what should be the appropriate type hint ? I'm not sure PHP handles that kind of union types... Not before version 8 anyway... Do you want me to change this ? Or maybe create another issue ?

Reply:
Since this is not the point of this PR, maybe we should wait for the merge THEN address this problem :x

Reply:
I suggest that we look at this in another PR?